### PR TITLE
feature: enable s3 prefix in waf logging

### PR DESCRIPTION
**Pull request by mayitbeegh**

### CODE REVIEW

File: src/main/java/com/nike/cerberus/command/core/CreateWafLoggingCommand.java [hunk 1]
```diff
@@ -45,6 +45,13 @@ public boolean isSkipStackCreation() {
         return skipStackCreation;
     }
 
+    @Parameter(names = {"--s3-prefix"}, description = "The prefix of WAF logs in S3 bucket.")
+    private String s3Prefix = "firehose/";
```
Comment (rdong333) [inline src/main/java/com/nike/cerberus/command/core/CreateWafLoggingCommand.java:49]: Does this mean the s3Prefix will always be `firehose/`?

Comment (mayitbeegh) [reply src/main/java/com/nike/cerberus/command/core/CreateWafLoggingCommand.java:49]: It defaults to firehose/ which is the current prefix https://github.com/Nike-Inc/cerberus-lifecycle-cli/blob/master/src/main/resources/cloudformation/waf-logging.yaml#L31
Just trying to make it backwards compatible

Comment (rdong333) [reply src/main/java/com/nike/cerberus/command/core/CreateWafLoggingCommand.java:49]: So if a prefix is provided `firehose/` would be overwritten?

Comment (mayitbeegh) [reply src/main/java/com/nike/cerberus/command/core/CreateWafLoggingCommand.java:49]: Correct

**Pull request merged**